pages/docs/user-docs/action-build.md

```diff
@@ -1,303 +1,185 @@
-    -l|--low       Specify low threshold (all checks, default) 
-    -m|--med       Perform medium and high checks
-    -h|--high      Perform only checks at level high
-
-See singularity check --help for available tags
+$ singularty build production2 development/
 ```
+You can convert the three supported container formats using any combination.
 
+Use care when converting writable ext3 images or sandbox directories to the default squashfs file format.  If changes were made to the writable container before conversion, there is no record of those changes in the Recipe file rendering your container non-reproducible.  It is a best practice to build your immutable production containers directly from a Recipe file instead.
 
-## Examples
-Let's quickly look at some definition file examples. These come straight from `singularity help build`. First, here
-is a recipe for build that serves only to describe the sections.
+## Building containers from Recipe files
+Of course, Recipe files can be used as the target when building a container.  For detailed information on writing Recipe files, please see [the Container Recipes docs](docs-recipes).
 
-```
-    %setup
-        echo "This is a scriptlet that will be executed on the host, as root, after"
-        echo "the container has been bootstrapped. To install things into the container"
-        echo "reference the file system location with $SINGULARITY_BUILDROOT"
-
-    %post
-        echo "This scriptlet section will be executed from within the container after"
-        echo "the bootstrap/base has been created and setup"
-
-    %test
-        echo "Define any test commands that should be executed after container has been"
-        echo "built. This scriptlet will be executed from within the running container"
-        echo "as the root user. Pay attention to the exit/return value of this scriptlet"
-        echo "as any non-zero exit code will be assumed as failure"
-        exit 0
-
-    %runscript
-        echo "Define actions for the container to be executed with the run command or"
-        echo "when container is executed."
-
-    %startscript
-        echo "Define actions for container to perform when started as an instance."
-
-    %labels
-        HELLO MOTO
-        KEY VALUE
-
-    %files
-        /path/on/host/file.txt /path/on/container/file.txt 
-        relative_file.txt /path/on/container/relative_file.txt
-
-    %environment 
-        LUKE=goodguy
-        VADER=badguy
-        HAN=someguy
-        export HAN VADER LUKE
+Let's say you already have the following Recipe file called `Singularity`, and you want to use it to build a container. 
 
 ```
+BootStrap: docker
+From: ubuntu:16.04
 
-Now let's look at a recipe that defines different <a href="https://containers-ftw.github.io/apps/" target="_blank">SCI-F apps</a>
+%post
+    apt-get -y update
+    apt-get -y install fortune cowsay lolcat
 
+%environment
+    export LC_ALL=C
+    export PATH=/usr/games:$PATH
 
```

Comment:
```
LC_ALL=C
PATH=/usr/games:$PATH
export LC_ALL PATH
```

Reply:
I think you are missing the point here.  There is no need for users to avoid bashisms in definition files.  It's not a best practice.  It's just pedantics that we enforce in the code so that Singularity can run on things like busybox.  I don't really want to start rewriting def file examples with syntax that many users would find awkward for no good reason.

Reply:
okay, that's ok with me then. 2 lines > 3!